src/asserts/signature-assert.js

```diff
@@ -37,7 +37,10 @@ export default function signatureAssert({ key, request } = {}) {
     const url = parse(`${protocol}://${host}${path}`, true);
 
     // Stringify body using sorted keys and encoding spaces as "+" instead of "%20".
-    const encoded = qs.stringify(body, { sort: (a, b) => a.localeCompare(b) }).replace(/%20/g, '+');
+    const encoded = qs.stringify(body, {
+      arrayFormat: 'brackets',
```

Comment:
Are you sure the returned values are using this method? Last time I checked it returned indexed ones.